src/main/java/com/iota/iri/network/protocol/Protocol.java

```diff
@@ -0,0 +1,197 @@
+package com.iota.iri.network.protocol;
+
+import com.iota.iri.controllers.TransactionViewModel;
+
+import java.nio.ByteBuffer;
+
+/**
+ * The IRI protocol uses a 4 bytes header denoting the version, type and length of a packet.
+ */
+public class Protocol {
+
+    /**
+     * The protocol version used by this node.
+     */
+    public final static byte PROTOCOL_VERSION = 1;
```

Comment:
I think we should leave this because then you can at a glance know what the highest supported protocol version of the given codebase is without having to look at the bitmasks.

Reply:
I now added exhaustive explanations of how the bitmasks function below.